EMBRACE_Client/MenuItemDataSource.h

```diff
@@ -9,9 +9,11 @@
 #import <Foundation/Foundation.h>
 #import "MenuItemImage.h"
 #import "PossibleInteraction.h"
+#import "Relationship.h"
 
 @interface MenuItemDataSource : NSObject {
     PossibleInteraction *interaction; //Stores the type of interaction, the object ids and the hotspots.
+    Relationship* menuRelationship;
```

Comment:
Why is Relationship being used here? Can't the information be extracted from PossibleInteraction?